src/core/Jalasoft.TeamUp.Projects.Core/Validators/InvitationValidator.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Validators
+{
+    using System;
+    using FluentValidation;
+    using Jalasoft.TeamUp.Projects.Models;
+
+    public class InvitationValidator : AbstractValidator<Invitation>
+    {
+        public InvitationValidator()
+        {
+            this.RuleFor(invitation => invitation.Id)
+                .NotEmpty().NotNull();
+
+            this.RuleFor(invitation => invitation.ProjectId)
+                .NotEmpty().NotNull();
+
+            this.RuleFor(invitation => invitation.ResumeId)
+                .NotEmpty().NotNull();
+
+            this.RuleFor(invitation => invitation.ProjectName)
+                .Length(3, 15)
+                .Matches("^[a-zñ A-ZÑ]+$")
+                .NotEmpty().NotNull();
+
+            this.RuleFor(invitation => invitation.ResumeName)
+                .Length(3, 15)
+                .Matches("^[a-zñ A-ZÑ]+$")
+                .NotEmpty().NotNull();
+
+            this.RuleFor(invitation => invitation.TextInvitation)
+                .MaximumLength(160);
+
+            this.RuleFor(invitation => invitation.Status)
+                .Matches("^[a-zñ A-ZÑ]+$");
```

Comment:
maybe an enumerable should be considered to have a defined list of possible statuses?

Reply:
.Matches("^(Invited|Rejected|Accepted)$");
This could be a better way